File: app/src/main/java/com/awscherb/cardkeeper/ui/scan/ScanActivity.java
package com.awscherb.cardkeeper.ui.scan;

import android.os.Bundle;
import android.support.annotation.Nullable;

import com.awscherb.cardkeeper.R;
import com.awscherb.cardkeeper.ui.base.BaseActivity;

public class ScanActivity extends BaseActivity {

    // ========================================================================
    // Lifecycle methods
    // ========================================================================

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        setContentView(R.layout.activity_scan);
        insertFragment(new ScanFragment());
    }
}

File: app/src/main/java/com/awscherb/cardkeeper/ui/scan/ScanActivity.kt
package com.awscherb.cardkeeper.ui.scan

import android.os.Bundle

import com.awscherb.cardkeeper.R
import com.awscherb.cardkeeper.ui.base.BaseActivity

class ScanActivity : BaseActivity() {

    // ========================================================================
    // Lifecycle methods
    // ========================================================================

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        setContentView(R.layout.activity_scan)
        insertFragment(ScanFragment())
    }
}
